proxy/proxy.go

```diff
@@ -1173,14 +1211,14 @@ func (p *Proxy) do(ctx *context, parentSpan ot.Span) (err error) {
 		ctx.ensureDefaultResponse()
 	} else if ctx.route.BackendType == eskip.LoopBackend {
 		loopCTX := ctx.clone()
-		loopSpan := tracing.CreateSpan("loopback", ctx.request.Context(), p.tracing.tracer)
+		_, loopSpan := p.tracing.Start(ctx.request.Context(), "loopback")
```

Comment:
What return value is ignored here and why?

Reply:
Same reason as here: https://github.com/zalando/skipper/pull/2578/files/a25b1840dc0decb0dfda88d2ca0163452eca73cd#r1542959056